subsys/mgmt/mcumgr/lib/cmd/fs_mgmt/src/fs_mgmt.c

```diff
@@ -159,11 +215,73 @@ fs_mgmt_file_download(struct mgmt_ctxt *ctxt)
 	return ok ? MGMT_ERR_EOK : MGMT_ERR_EMSGSIZE;
 }
 
+static int fs_mgmt_write(const char *path, size_t offset,
```

Comment:
Nothing wrong with these functions, but, the zephyr_fs_mgmt.c file does not appear to have been removed in this commit when it should be